html/workout.html

```diff
@@ -7,28 +7,170 @@
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato">
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Raleway">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+
     <title>Workout - Motivation Co.</title>
 </head>
 
 <body>
     <!--NAVIGATION BAR BEGINS-->
     <header>
+
         <div class="nav-bar">
+
             <div class="left-nav-bar">
-                <a href="../index.html" style="text-decoration: none;"><h2>Motivation Co.</h2></a>
+                <a href="index.html" style="text-decoration: none;"><h2>Motivation Co.</h2></a>
                 <img src="../imgs/gearicon.png" width="25" height="25" alt="gear icon">
             </div>
             <nav>
                 <ul>
-                    <li><a href="../index.html">Home</a></li>
+                    <li><a href="index.html">Home</a></li>

```

Comment:
Keep ../ reference. Also add about page.
```suggestion
                    <li><a href="../index.html">Home</a></li>
                    <li><a href="about.html">About</a></li>
```